execution/executor-benchmark/src/main.rs

```diff
@@ -12,11 +12,17 @@ static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
 
 #[derive(Debug, StructOpt)]
 struct PrunerOpt {
+    #[structopt(long)]
+    enable_state_store_pruner: bool,
+
+    #[structopt(long)]
+    enable_ledger_pruner: bool,
+
     #[structopt(long, default_value = "100000", help = "Set to -1 to disable.")]
-    state_prune_window: i64,
+    state_prune_window: u64,
```

Comment:
We should only enable this flag when the corresponding boolean is true.

use "requires", e.g.:
https://github.com/aptos-labs/aptos-core/blob/68215374a17c99cf9f90dff273a309f7f35017ae/crates/transaction-emitter-lib/src/args.rs#L87

Reply:
Done.